plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/model/sql/generator/SQLGeneratorProcedureCheck.java

```diff
@@ -32,19 +32,11 @@
 public class SQLGeneratorProcedureCheck extends SQLGeneratorProcedure {
 
     /**
-     * Generate PostgreSQL procedure check SQL entry point
+     * Generate PostgreSQL procedure check SQL - via https://github.com/okbob/plpgsql_check
      */
     @Override
     protected void generateSQL(DBRProgressMonitor monitor, StringBuilder sql, DBSProcedure proc) throws DBException {
-        Collection<? extends DBSProcedureParameter> parameters = proc.getParameters(monitor);
-        generateStoredProcedureCall(sql, proc, CommonUtils.safeCollection(parameters));
-    }
-    
-    /**
-     * Generate PostgreSQL procedure check SQL - via https://github.com/okbob/plpgsql_check
-     */
-    private void generateStoredProcedureCall(StringBuilder sql, DBSProcedure proc, 
-            Collection<? extends DBSProcedureParameter> parameters) {
+        Collection<? extends DBSProcedureParameter> parameters = CommonUtils.safeCollection(proc.getParameters(monitor));
         List<DBSProcedureParameter> inParameters = new ArrayList<>();
         inParameters.addAll(CommonUtils.safeCollection(parameters));
```

Comment:
You don't need these. Just inline `CommonUtils.safeCollection(proc.getParameters(monitor))` in the loop, like so:

```java
for (DBSProcedureParameter parameter : CommonUtils.safeCollection(proc.getParameters(monitor))) {
    ...
}
```